backend/src/modules/lessons/router.js

```diff
@@ -0,0 +1,125 @@
+import { Router } from 'express'
+import { checkSchema, validationResult } from 'express-validator'
+
+import Controller from './controller'
+import checkRoles from '../../middlewares/checkRoles'
+import verifyUser from '../../middlewares/verifyUser'
+
+const router = Router()
+
+router.use(verifyUser)
+router.use(checkRoles(['teacher']))
+
+router.post('/create', checkSchema({
+  classID: {
+    in: 'body',
+    isNumeric: {
+      errorMessage: 'ClassID should be numeric',
+    },
+    notEmpty: {
+      errorMessage: 'ClassID shouldn`t be empty',
+    },
+  },
+  date: {
+    in: 'body',
+    isISO8601: {
+      errorMessage: 'Invalid date',
+    },
+    notEmpty: {
+      errorMessage: 'Date shouldn`t be empty',
+    },
+  },
+  topic: {
+    in: 'body',
+    isString: {
+      errorMessage: 'Invalid topic',
+    },
+    notEmpty: {
+      errorMessage: 'Topic shouldn`t be empty',
+    },
+  },
+  home_work: {
```

Comment:
So, here you're validating home_work, but in Controller.create, you use homeWork. Don't forget to change and check everything, 'cause this just won't work.